src/component/menu/ApiDetails/ApiDetails.tsx

```diff
@@ -0,0 +1,106 @@
+import useUiConfig from '../../../hooks/api/getters/useUiConfig/useUiConfig';
+import { ReactElement } from 'react';
+import ConditionallyRender from '../../common/ConditionallyRender';
+import { IVersionInfo } from '../../../interfaces/uiConfig';
+
+interface IPartialUiConfig {
+	name: string;
+	version: string;
+	slogan?: string;
+	environment?: string;
+	versionInfo?: IVersionInfo;
+}
+
+interface IProps {
+	uiConfig: IPartialUiConfig;
+}
+
+export function ApiDetailsContainer(): ReactElement {
+	const { uiConfig } = useUiConfig();
+
+	return <ApiDetails uiConfig={uiConfig} />;
+}
+
+export function ApiDetails(props: IProps): ReactElement {
```

Comment:
We're trying to follow a convention of using the arrow function pattern when it comes to components, just to keep it consistent across the repository: 

```
const ApiDetails = () => {
    ...
}
```